package.json

```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
     "build": "npm run build:lib",
-    "build:lib": "babel src --out-dir lib",
+    "build:lib": "babel src --out-dir lib && uglifyjs ./lib/index.js -o ./lib/index.min.js",
```

Comment:
Is this necessary for a module? Webpack takes care of this for production builds